GNUmakefile

```diff
@@ -42,7 +42,7 @@ websitefmtcheck:
 lint:
 	@echo "==> Checking source code against linters..."
 	# https://github.com/golangci/golangci-lint/issues/337 fixing error
-	golangci-lint run ./$(PKG_NAME) -v --deadline=30m
+	bin/golangci-lint run ./$(PKG_NAME) -v --deadline=30m
```

Comment:
drive-by: golangci-lint is installed under `bin/` - there's no point relying on the tool being available in the system path. cc @gssbzn in case I'm missing something

Reply:
line 16 includes the bin folder in the `PATH` (`export PATH := ./bin:$(PATH)`) for convenience, no need to revert this just curious if that no longer works?

Reply:
Didn't work for me on Mac, but it worked in GH actions